packages/nitro-protocol/test/contracts/ForceMove/checkpoint.test.ts

```diff
@@ -93,8 +93,8 @@ describe('checkpoint', () => {
   it.each`
     description | largestTurnNum                         | support              | finalizesAt  | reason
     ${accepts1} | ${turnNumRecord + 1}                   | ${valid}             | ${undefined} | ${undefined}
-    ${accepts2} | ${turnNumRecord + 1}                   | ${valid}             | ${never}     | ${undefined}
-    ${accepts3} | ${turnNumRecord + 1 + participantsNum} | ${valid}             | ${future}    | ${undefined}
+    ${accepts2} | ${turnNumRecord + 1}                   | ${valid}             | ${future}    | ${undefined}
+    ${accepts3} | ${turnNumRecord + 1 + participantsNum} | ${valid}             | ${never}     | ${undefined}
```

Comment:
The description of `accepts2` test case states a challenge must be in progress, so `finalizesAt` should be `future`.